test/testdata/deployednettemplates/recipes/devnet/genesis.json

```diff
@@ -0,0 +1,113 @@
+{
+	"NetworkName": "",
+	"VersionModifier": "",
+	"ConsensusProtocol": "",
+	"FirstPartKeyRound": 0,
+	"LastPartKeyRound": 3000000,
```

Comment:
I kind of agree with Tsachi; I think having a higher count (e.g. 10,000,000) would be nice here. We can always rotate keys early for practice, and it gives us a little more time. Not required for this, though.

Reply:
OK, adjusting to 10,000,000.

Reply:
Restarted network with the id of v1.0 and participation key expiration set to 10,000,000 rounds.